package.json

```diff
@@ -25,35 +25,28 @@
         "ocamlfind",
         "sexplib"
       ]
-    },
-    "buildsInSource": "_build"
+    }
   },
   "dependencies": {
-    "@opam/alcotest": "0.8.3",
-    "@opam/astring": "0.8.3",
-    "@opam/bisect_ppx": "1.3.4",
-    "@opam/bos": "0.2.0",
-    "@opam/cmdliner": "1.0.2",
-    "@opam/cppo": "1.6.5",
-    "@opam/dune": "1.3.0",
-    "@opam/fpath": "0.7.2",
-    "@opam/js-build-tools": "113.33.04",
-    "@opam/lambdasoup": "0.6.3",
-    "@opam/markup": "*",
-    "@opam/oasis": "0.4.11",
-    "@opam/ocamlfind": "1.8.0",
-    "@opam/result": "1.3",
-    "@opam/sexplib": "*",
-    "@opam/tyxml": "4.3.0",
-    "ocaml": "~4.2.0"
-  },
-  "resolutions": {
-    "@opam/markup": "aantron/markup.ml:markup.opam#9f8e77",
-    "@opam/sexplib": "bryphe/sexplib#548f94d"
+    "@opam/bisect_ppx": "^1.3.4",
+    "@opam/bos": "*",
+    "@opam/dune": "^1.3.0",
+    "@opam/tyxml": "^4.3.0",
+    "ocaml": "*"
   },
   "devDependencies": {
+    "@opam/alcotest": "^0.8.3",
+    "@opam/astring": "^0.8.3",
+    "@opam/cmdliner": "^1.0.2",
+    "@opam/cppo": "^1.6.5",
+    "@opam/fpath": "^0.7.2",
+    "@opam/markup": ">= 0.8.0",
     "@opam/merlin": "^3.0.3",
-    "ocaml": "~4.2.0"
+    "@opam/ocamlfind": "^1.8.0",
+    "@opam/result": "*",
+    "@opam/rresult": "*",
+    "@opam/sexplib": "*",
```

Comment:
We had a problem with sexplib on 4.02 previously, see https://github.com/ocaml/odoc/pull/216#issuecomment-430823206. Solving it resulted in the resolution entry, and also dependencies on OASIS and js-build-tools. This `*` constraint looks like it might cause a regression.

Reply:
I tested this on windows and it built, but I built with a later compiler, will test again to see if it has been fixed upstream.

Reply:
I tested by setting ocaml to `~4.2.0` and sexplib to the same as in the opam file `>= 113.33.00` but then it couldn't find versions that were okay.
The resolution we had before is broken as there is not manifest. When I update to the latest commit we get a dependency in `sexplib0` that needs `>4.4` of the compiler.

Would a good trade-off be that we don't need to support down to `4.2` when using esy?

Reply:
@ulrikstrid if we are to build binaries to be used with BuckleScript `.cmt` files, we need to support building on `4.02`

Reply:
@ostera Will these be built with `esy`? And as BuckleScript is actively working in switching to `4.06`, is this a issue?

Reply:
If BuckleScript actually moves to `4.06` then sure, but until that happens (and for everyone that's isn't running the latest release when that happens), we should support `4.02`.

Ideally the odoc binaries would be built with `esy` too, yes 👍

Reply:
I might have come at this from the wrong angle. I used this package.json when linking from another project to test out a PR I was working on. But you probably want this package.json to just build binaries and don't care about dependants as they should depend on the opam file right?